actionview/lib/action_view/helpers/asset_tag_helper.rb

```diff
@@ -113,14 +127,22 @@ def javascript_include_tag(*sources)
       def stylesheet_link_tag(*sources)
         options = sources.extract_options!.stringify_keys
         path_options = options.extract!("protocol", "host", "skip_pipeline").symbolize_keys
-        sources.uniq.map { |source|
+        early_hints_links = []
+
+        sources_tags = sources.uniq.map { |source|
+          href = path_to_stylesheet(source, path_options)
+          early_hints_links << "<#{href}>; rel=preload; as=stylesheet"
```

Comment:
According to the example described in https://w3c.github.io/preload/#as-attribute, it seems that the `as` target attribute for stylesheets should be `style` instead of `stylesheet`